ui/blocks/BottomBar.tsx

```diff
@@ -39,6 +39,13 @@ const BottomBar: React.FC<BottomBarProps> = ({ currentPath }) => {
         active={currentPath === '/info'}
         color="red"
       />
+      <Tab
```

Comment:
The tabs have flexBasis of 33.3%, so the fourth tab breaks the page layout a bit :D Before merging it should be removed, and accordingly we would need a solution where to put the link to the qr code. I was thinking we could maybe just add a few buttons to index.tsx, as quicklinks to e.g. the Owner Registration/Login – that would also be a good place to link to the qr scanner

Reply:
Remove that button when it's not needed. I don't think it's necessary to have it at all (when you're in the app then you're in the app).